db/download/Makefile

```diff
@@ -1,8 +1,6 @@
-# Makefile to run the Python script
-
-.PHONY: all download
-
+.PHONY: all
```

Comment:
If you're gonna use `all` to work on the `download` function, it may be easier to use:
```make
default: download

.PHONY: download
download:
	python3 src/src.py
```

This way, you just need to enter `make` and it'll run `make download`.